src/app/route/MyAccountDetails/MyAccountDetails.component.js

```diff
@@ -298,13 +310,17 @@ class MyAccountDetails extends Component {
                           validation={ ['notEmpty'] }
                           value={ region && region.region }
                         />
-                        <Field
-                          type="text"
-                          label="Country"
-                          id="country_id"
-                          validation={ ['notEmpty'] }
-                          value={ country_id }
-                        />
+                        <div block="Field">
+                            <span block="MyAccountDetails" elem="Country">Country</span>
```

Comment:
Maybe we should put select in Field component? And use one interface for all inputs? I mean on Change... Not sure. Dislike labeling every select by hand.

Reply:
Select is now part of Field. Can be used independently also (see: CategoryPage).